packages/dashboard/src/components/sidePanel/sections/inputSettingsSection/index.tsx

```diff
@@ -20,9 +20,19 @@ const InputSettings: FC<InputComponentProps> = ({ messageOverride }) => {
   const {
     sidePanel: { inputSettings },
   } = messageOverride;
-  const [options, setOptions] = useInputWidgetInput('options');
+  // TODO: Add type support for nested paths in useInput
+  // eslint-disable-next-line
+  // @ts-ignore
+  const useInputOptions = useInputWidgetInput('properties.options');
```

Comment:
a bit confusing to refer to the return of a hook as 'use___' as the use prefix is reserved for the hook itself, not it's return statement. The more standard way to writ this would be `const [options, setOptions] = usesInputWidgetInput...`

Reply:
Good callout. That was my first approach yet with nested props that throws type errors. Let me try to find to find a better way of doing this.